src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -42,6 +53,7 @@ class AuthApplicationSection extends React.Component {
 
     return (
       <div className="profile-section medium-12 columns">
+        {hasVerifiedSavedForms && <FormList className="sip-form-list" handleClick={this.handleClick} savedForms={verifiedSavedForms}/>}
```

Comment:
Is the `className` prop used?

Reply:
No, good catch, the styles now feature classes at the lowest possible levels per your comments elsewhere regarding nesting, it will be removed.